src/main/java/com/aws/greengrass/util/Exec.java

```diff
@@ -347,10 +347,9 @@ public Exec withErr(Consumer<CharSequence> o) {
     /**
      * Create and start the child process in platform-specific ways.
      *
-     * @return child process
      * @throws IOException if IO error occurs
      */
-    protected abstract Process createProcess() throws IOException;
```

Comment:
Why are you changing this? What's the point in breaking compatibility?

Reply:
it seems redundant to do process=winPb.start(), then return process and assign it again with process=createProcess()

Reply:
Keep it the way that it is. The API is more clear that way and it enforce the requirement that `Exec.process` is properly set to the process object. With the changed API you are relying on the implementor to set the field correctly which they may not do because there's no obvious need to do that.